bundle/edu.gemini.sp.vcs/src/main/scala/edu/gemini/sp/vcs2/MergePlan.scala

```diff
@@ -3,6 +3,7 @@ package edu.gemini.sp.vcs2
 import edu.gemini.pot.sp._
 import edu.gemini.pot.sp.version._
 import edu.gemini.shared.util._
+import edu.gemini.shared.util.IntegerIsIntegral._
```

Comment:
It all starts from a warning given by scalac, that you shouldn't have top-level constructs like `IntegerIsIntegral` in a package object. The fix seems simple, just move it out but then you loose the implicit that exposes `IntegerIsIntegral`. My solution (which maybe far from perfect) was to move it out of the package object and add an object `IntegerIsIntegral` with the implicit conversion